RecoTracker/MkFitCMS/src/MkStdSeqs.cc

```diff
@@ -399,54 +429,62 @@ namespace mkfit {
       const auto ntracks = tracks.size();
 
       std::vector<float> ctheta(ntracks);
+      std::multimap<int, int> hitMap;
+
       for (auto itrack = 0U; itrack < ntracks; itrack++) {
         auto &trk = tracks[itrack];
         ctheta[itrack] = 1.f / std::tan(trk.theta());
+        for (int i = 0; i < trk.nTotalHits(); ++i) {
+          if (trk.getHitIdx(i) < 0)
+            continue;
+          int a = trk.getHitLyr(i);
+          int b = trk.getHitIdx(i);
+          hitMap.insert(std::make_pair(b * 1000 + a, i > 0 ? itrack : -itrack));  //negative for first hit in trk
```

Comment:
what's 1000?

Reply:
a way to write layer and hit id in a single int.

Reply:
I know, I commented already here: https://github.com/cms-sw/cmssw/pull/36546#discussion_r803464227, not sure if it's in the plans to document it.